astropy/coordinates/funcs.py

```diff
@@ -149,11 +152,23 @@ def get_sun(time):
     from .builtin_frames import GCRS
 
     earth_pv_helio, earth_pv_bary = erfa.epv00(time.jd1, time.jd2)
-    x = -earth_pv_helio[..., 0, 0] * u.AU
-    y = -earth_pv_helio[..., 0, 1] * u.AU
-    z = -earth_pv_helio[..., 0, 2] * u.AU
+
+    # We have to manually do abberation because we're outputting directly into
```

Comment:
Yeah, that make senses - there's a builtin_frames/utils.py that has helper functions used in some of the other frames. 

I'll give it some time to see if there are other comments on this, but it's easy enough to pull this out and put it there.